### PR TITLE
Changelog v1.12.3

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.12.3](https://github.com/go-gitea/gitea/releases/tag/v1.12.3) - 2020-07-28
+
+* BUGFIXES
+  * Don't change creation date when updating Release (#12351)
+  * Show 404 page when release not found (#12328) (#12332)
+  * Backport emoji fixes to 1.12 (#12327)
```
Comment (techknowlogick) [inline CHANGELOG.md]: This is two backports in one, could you split this line to both?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.12.3](https://github.com/go-gitea/gitea/releases/tag/v1.12.3) - 2020-07-28
+
+* BUGFIXES
+  * Don't change creation date when updating Release (#12351)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Could you also reference original PR #12343

**Review by techknowlogick — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by techknowlogick** [comment]
@6543 thanks for PR :) please send frontport

**Comment by 6543** [comment]
frontport: https://github.com/go-gitea/gitea/pull/12357

**Pull request merged**